viewer/src/views/threejs/cad/CadNode.ts

```diff
@@ -182,16 +186,19 @@ export class CadNode extends THREE.Object3D {
     const currentLevelOfDetail = new Map<number, LevelOfDetail>();
     // Do not request sectors that are already in scene
     const filterWantedSectorsDistinctFromCurrent = filter(
-      (wantedSector: WantedSector) => currentLevelOfDetail.get(wantedSector.id) !== wantedSector.levelOfDetail
+      (wantedSector: WantedSector) => currentLevelOfDetail.get(wantedSector.sectorId) !== wantedSector.levelOfDetail
     );
     // Do not continue with sectors already in scene
     const filterConsumedSectorsDistinctFromCurrent = filter(
-      (consumedSector: ConsumedSector) => currentLevelOfDetail.get(consumedSector.id) !== consumedSector.levelOfDetail
+      (consumedSector: ConsumedSector) =>
+        currentLevelOfDetail.get(consumedSector.sectorId) !== consumedSector.levelOfDetail
     );
     const pipeline = new Subject<ThreeCameraConfig>();
     pipeline
       .pipe(
-        auditTime(100),
+        // TODO 2020-04-15 larsmoa: Reduce delay to something more sensible
+        // Temporary workaround to avoid flooding the GPU pipeline with readPixels.
+        debounceTime(150),
```

Comment:
Will not the change to `debounceTime` cause us to potentially never update the sectors? That is, if you continue to move the camera, debounceTime will just postpone and postpone updating forever, leaving us with nothing on screen. Why not just increase the time sent to `auditTime`?

Reply:
Yes, it will. I considered it the lesser evil of causing stuttering while moving around. Because we load geometry for the full model at all times (just at low detail), I considered acceptable for now. I realize it's far from optimal, but I don't see a good solution right now and the upside of the new culler outbeats the disadvantage this results in.

Note! A theory I have is that we should not do `render(); readPixels();` in the same call to `determineSectors()` as this will cause the render-pipeline to flush before `readPixels` returns.